src/plugins/data/common/field_formats/converters/percent.ts

```diff
@@ -17,29 +17,28 @@
  * under the License.
  */
 
-import { NumeralFormat } from './numeral';
-import { TextContextTypeConvert, FIELD_FORMAT_IDS } from '../types';
+import { i18n } from '@kbn/i18n';
+import { IntlNumberFormat } from './intl_number_format';
+import { FIELD_FORMAT_IDS } from '../types';
 
-export class PercentFormat extends NumeralFormat {
+export class PercentFormat extends IntlNumberFormat {
   static id = FIELD_FORMAT_IDS.PERCENT;
-  static title = 'Percentage';
+  static title = i18n.translate('data.common.fieldFormats.percent.title', {
+    defaultMessage: 'Percent',
+  });
 
   id = PercentFormat.id;
   title = PercentFormat.title;
   allowsNumericalAggregations = true;
 
   getParamDefaults = () => ({
-    pattern: this.getConfig!('format:percent:defaultPattern'),
-    fractional: true,
+    minDecimals: 0,
+    maxDecimals: 3,
```

Comment:
I wonder if 3 is the right default for percentage? I almost think maybe 0 would be the right default, but maybe not.

Reply:
I think I will keep it at 3 because it's the minimum possible change- 3 was the previous default.